src/ONYXKEYS.ts

```diff
@@ -375,8 +376,9 @@ type OnyxValues = {
     // Collections
     [ONYXKEYS.COLLECTION.DOWNLOAD]: OnyxTypes.Download;
     [ONYXKEYS.COLLECTION.POLICY]: OnyxTypes.Policy;
-    [ONYXKEYS.COLLECTION.POLICY_CATEGORIES]: unknown;
+    [ONYXKEYS.COLLECTION.POLICY_CATEGORIES]: OnyxTypes.PolicyCategory;
     [ONYXKEYS.COLLECTION.POLICY_MEMBERS]: OnyxTypes.PolicyMember;
+    [ONYXKEYS.COLLECTION.POLICY_RECENTLY_USED_CATEGORIES]: OnyxTypes.RecentlyUsedCategories;
```

Comment:
AFAIK, all types inside `COLLECTION` is being converted like `Record<string, T>`. It means we don't have a simple array here. @fabioh8010, am I right?

Reply:
Associated with - https://github.com/Expensify/App/pull/26501#discussion_r1315693505.

Reply:
Yes, all onyx keys inside `COLLECTION` will be represented as a record of items of that type, e.g. `OnyxTypes.RecentlyUsedCategories` will be `Record<string, OnyxTypes.RecentlyUsedCategories | null>`

Reply:
@yuwenmemon, so based on this, I think we should move recently used categories out of `COLLECTION`. Ideas?

Reply:
I think there's some confusion over the double naming. This is the RecentlyUsedCategories type:

https://github.com/Expensify/App/blob/cad29e52f92ad1f939adc9e60f11febf79053cd6/src/types/onyx/RecentlyUsedCategories.ts#L1-L3

It's an array of strings. 

> Yes, all onyx keys inside COLLECTION will be represented as a record of items of that type, e.g. OnyxTypes.RecentlyUsedCategories will be Record<string, OnyxTypes.RecentlyUsedCategories | null>

This is exactly what we want and what we're getting:
<img width="1324" alt="266169494-8278abd4-b069-43fd-8596-84abe066d3ba" src="https://github.com/Expensify/App/assets/4741899/a8c52997-9e37-406a-bd4e-6d958ddb0f7f">

Reply:
Yes, makes sense.